docs/packaging_policy.md

```diff
@@ -108,29 +108,62 @@ for each combination. There are some particular cases for this general rule:
+By default recipes should use `shared=False` with `fPIC=True`. If support, `header_only=False` is the default.
+
+Usage of each option should follow the rules:
+
+* `shared` (with values `True` or `False`). The CI inspects the recipe looking for this option. The **default should be `shared=False`** and will
    generate all the configurations with values `shared=True` and `shared=False`.
 
-   > Note.- The CI applies `shared=True` only to the package being built, while every other requirement will use their defaults
-   > (typically `shared=False`). It's important to keep this in mind when trying to consume shared packages from ConanCenter
+   > **Note**: The CI applies `shared=True` only to the package being built, while every other requirement will `shared=False`. To consume everything as a shared library you will set `--build=always` and/or `-o *:shared=True`)
+   > It's important to keep this in mind when trying to consume shared packages from ConanCenter
    > as their requirements were linked inside the shared library. See [FAQs](faqs.md#how-to-consume-a-graph-of-shared-libraries) for more information.
 
-* `header_only` (with values `True` or `False`). If the CI detects this option, it will generate all the configurations for the
-   value `header_only=False` and add one more configuration with `header_only=True`. **Only one
-   package** will be generated for `header_only=True`, so it is crucial that the package is actually a _header only_ library, with header files only (no libraries or executables inside).
+* `fPIC` (with values `True` or `False`). The **default should be `fPIC=True`** and will generate all the configurations with values `fPIC=True` and `fPIC=False`.
+  This option does not make sense on all the support configurations so it should be removed.
+
+   ```python
+   def config_options(self):
+       if self.settings.os == "Windows":
+      del self.options.fPIC
+
+   def configure(self):
+      if self.options.shared:
+         try:
+             del self.options.fPIC
+         except Exception:
+             pass
+   ```
+
+   Starting with Conan 1.53 this can be simplified to
+
+   ```py
+   def configure(self):
+      if self.options.shared:
+         self.options.rm_safe("fPIC")
+   ```
+
+* `header_only` (with values `True` or `False`). The **default should be `header_only=False`**. If the CI detects this option, it will generate all the
+   configurations for the value `header_only=False` and add one more configuration with `header_only=True`. **Only one package**
+   will be generated for `header_only=True`, so it is crucial that the package is actually a _header only_ library, with header files only (no libraries or executables inside).
 
    Recipes with such option should include the following in their `package_id` method
 
    ```python
    def package_id(self):
       if self.options.header_only:
-         self.info.header_only()
+         self.info.clear()
```

Comment:
It would be nice adding a note about how it affects `self.info.settings` and `self.info.options`. Settings is None and `settings.compiler` is not make affect for `check_min_cppstd`, only when installing the package. Otherwise, it would require `validate_build` with `self.settings.compiler`

Reply:
I think that should go into the main Conan docs since it's pretty generic.
"For more information about how this affects the `package_id` read the documentation [here]()" type of thing?